chore(coingecko)_: request data from coingecko provider by token key (coingecko id) instead of symbol

saledjenic opened the pull request:
Up until now, coingecko requested the data using the symbol. Since there can be more tokens with the same symbol, the only correct way is to use a chain+token-address pair. But since we want to group(aggregate) tokens by name across chains and since coingecko's tokens list already sets the id parameter for each token in their list, which is guaranteed to be unique, we've decided to support coingecko's tokens list only.

What's changed:
- coingecko provider updated to use grouped tokens key (coingecko tokens' id) instead of symbol
- aave and uniswap token lists removed
- status token list contains testnet tokens only, updated with id that matches coingecko tokens' id
- `marketManager` works with coingecko provider only (for now, until we add coin-gecko to the proxy), `cryptoCompare` and `cryptoCompareProxy` providers are removed as options
- cryptocompare providers' code removed

An important improvement:
- for the difference from before, the market provider doesn't fetch tokens separately, but instead uses the tokens fetched and managed by the `token` package, which ensures using the same list of tokens everywhere in the app

Changed endpoints (all of them accept list of grouped tokens key/s (which is actually coingecko tokens' id) instead list of symbol/s):
- `wallet_fetchPrices`
- `wallet_fetchMarketValues`
- `wallet_getHourlyMarketValues`
- `wallet_getDailyMarketValues`
- `wallet_fetchTokenDetails`
- `wallet_fetchAllCurrencyFormats`
- `wallet_getCachedCurrencyFormats`

Changed signals:
- `UpdatedPrices` of `wallet.suggested.routes` signal returns price/s per grouped tokens key/s

----
Initially, this PR https://github.com/status-im/status-go/pull/6486 was created that handles the token key as `chainId-address` pair, but as explained in this comment we closed it https://github.com/status-im/status-go/pull/6486#issuecomment-2772068230.

saledjenic committed:
chore(coingecko)_: request data from coingecko provider by token key (coingecko id) instead of symbol

Up until now the data were requested by coingecko using the symbol, but since there can be more
tokens with the same symbol, the only correct way is to use chain+token-address pair. But since we want
to gorup(aggregate) tokens by name across chains and since coingecko tokens list already setup the id
parameter for each token in their list, which is guaranteed to be unique, we've decided to support coingecko
tokens list only.

- `id` param exposed via `Token` type through the api and should be used for aggregation on a client side.

saledjenic committed:
chore(wallet)_: token lists updated to use coingecko (for mainnet tokens) and status (for testnet tokens) tokens lists

- aave and uniswap token lists removed
- status token list contains testnet tokens only, updated with id that matches coingecko tokens' ids

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (96)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/1/) | 2025-04-02 09:40:43 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250402-093728-aa9d6a-pr6497.aar) |
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/1/) | 2025-04-02 09:41:00 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250402-093730-aa9d6a-pr6497.zip) |
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/1/) | 2025-04-02 09:42:15 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250402-093739-aa9d6a-pr6497.zip) |
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/1/) | 2025-04-02 09:42:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250402-093733-aa9d6a-pr6497.zip) |
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/1/) | 2025-04-02 09:42:59 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250402-093735-aa9d6a-pr6497.zip) |
| :heavy_check_mark: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/1/) | 2025-04-02 09:43:29 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250402-093731-aa9d6a-pr6497.zip) |
| :heavy_multiplication_x: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/1/) | 2025-04-02 09:47:52 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/1//consoleText) |
| :heavy_multiplication_x: | aa9d6a93 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/1/) | 2025-04-02 10:14:14 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/2/) | 2025-04-02 09:50:03 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250402-094654-c98171-pr6497.aar) |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/2/) | 2025-04-02 09:50:16 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250402-094703-c98171-pr6497.zip) |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/2/) | 2025-04-02 09:50:21 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250402-094655-c98171-pr6497.zip) |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/2/) | 2025-04-02 09:52:07 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250402-094658-c98171-pr6497.zip) |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/2/) | 2025-04-02 09:52:18 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250402-094657-c98171-pr6497.zip) |
| :heavy_check_mark: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/2/) | 2025-04-02 09:52:19 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250402-094700-c98171-pr6497.zip) |
| :heavy_multiplication_x: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/2/) | 2025-04-02 09:56:00 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/2//consoleText) |
| :heavy_multiplication_x: | c9817123 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/2/) | 2025-04-02 10:48:56 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/3/) | 2025-04-03 07:50:16 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250403-074711-9151b4-pr6497.aar) |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/3/) | 2025-04-03 07:50:41 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250403-074712-9151b4-pr6497.zip) |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/3/) | 2025-04-03 07:50:57 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250403-074720-9151b4-pr6497.zip) |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/3/) | 2025-04-03 07:52:28 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250403-074714-9151b4-pr6497.zip) |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/3/) | 2025-04-03 07:52:47 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250403-074716-9151b4-pr6497.zip) |
| :heavy_check_mark: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/3/) | 2025-04-03 07:54:09 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250403-074713-9151b4-pr6497.zip) |
| :heavy_multiplication_x: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/3/) | 2025-04-03 07:57:02 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/3//consoleText) |
| :heavy_multiplication_x: | 9151b497 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/3/) | 2025-04-03 08:24:41 | ~37 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/4/) | 2025-04-04 06:56:51 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250404-065342-68a8a8-pr6497.aar) |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/4/) | 2025-04-04 06:57:13 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250404-065343-68a8a8-pr6497.zip) |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/4/) | 2025-04-04 06:57:21 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250404-065351-68a8a8-pr6497.zip) |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/4/) | 2025-04-04 06:59:04 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-065346-68a8a8-pr6497.zip) |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/4/) | 2025-04-04 06:59:18 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-065347-68a8a8-pr6497.zip) |
| :heavy_check_mark: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/4/) | 2025-04-04 07:01:02 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250404-065345-68a8a8-pr6497.zip) |
| :heavy_multiplication_x: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/4/) | 2025-04-04 07:03:14 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/4//consoleText) |
| :heavy_multiplication_x: | 68a8a8ad | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/4/) | 2025-04-04 07:29:20 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/5/) | 2025-04-04 08:41:32 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250404-083813-89fc07-pr6497.aar) |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/5/) | 2025-04-04 08:41:47 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250404-083815-89fc07-pr6497.zip) |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/5/) | 2025-04-04 08:41:47 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250404-083823-89fc07-pr6497.zip) |
| :heavy_multiplication_x: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/5/) | 2025-04-04 08:42:32 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/5//consoleText) |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/5/) | 2025-04-04 08:43:35 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-083817-89fc07-pr6497.zip) |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/5/) | 2025-04-04 08:43:41 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250404-083815-89fc07-pr6497.zip) |
| :heavy_check_mark: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/5/) | 2025-04-04 08:43:50 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-083819-89fc07-pr6497.zip) |
| :heavy_multiplication_x: | 89fc07b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/5/) | 2025-04-04 08:46:41 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/6/) | 2025-04-04 10:34:14 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250404-103101-6d0bba-pr6497.zip) |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/6/) | 2025-04-04 10:34:22 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250404-103109-6d0bba-pr6497.zip) |
| :heavy_multiplication_x: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/6/) | 2025-04-04 10:34:32 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/6//consoleText) |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/6/) | 2025-04-04 10:34:32 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250404-103100-6d0bba-pr6497.aar) |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/6/) | 2025-04-04 10:34:37 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-103105-6d0bba-pr6497.zip) |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/6/) | 2025-04-04 10:36:27 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250404-103105-6d0bba-pr6497.zip) |
| :heavy_check_mark: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/6/) | 2025-04-04 10:37:40 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250404-103102-6d0bba-pr6497.zip) |
| :heavy_multiplication_x: | 6d0bba66 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/6/) | 2025-04-04 10:39:31 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/7/) | 2025-04-07 06:34:24 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250407-063105-091381-pr6497.aar) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/7/) | 2025-04-07 06:34:32 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250407-063105-091381-pr6497.zip) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/7/) | 2025-04-07 06:36:32 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-063112-091381-pr6497.zip) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/7/) | 2025-04-07 06:37:23 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-063111-091381-pr6497.zip) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/7/) | 2025-04-07 06:37:41 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250407-063117-091381-pr6497.zip) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/7/) | 2025-04-07 06:38:30 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250407-063107-091381-pr6497.zip) |
| :heavy_multiplication_x: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/7/) | 2025-04-07 06:40:11 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/7//consoleText) |
| :heavy_check_mark: | 0913810e | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/7/) | 2025-04-07 07:07:09 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/8/) | 2025-04-07 06:37:39 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250407-063428-f8623f-pr6497.aar) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/8/) | 2025-04-07 06:37:57 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250407-063436-f8623f-pr6497.zip) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/8/) | 2025-04-07 06:41:29 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-063729-f8623f-pr6497.zip) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/8/) | 2025-04-07 06:41:54 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-063637-f8623f-pr6497.zip) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/8/) | 2025-04-07 06:42:02 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250407-063744-f8623f-pr6497.zip) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/8/) | 2025-04-07 06:44:03 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250407-063834-f8623f-pr6497.zip) |
| :heavy_multiplication_x: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/8/) | 2025-04-07 06:48:16 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/8//consoleText) |
| :heavy_check_mark: | f8623f72 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/8/) | 2025-04-07 07:40:52 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/9/) | 2025-04-07 16:19:02 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250407-161545-ea571c-pr6497.aar) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/9/) | 2025-04-07 16:19:12 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250407-161554-ea571c-pr6497.zip) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/9/) | 2025-04-07 16:19:13 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250407-161546-ea571c-pr6497.zip) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/9/) | 2025-04-07 16:21:00 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-161549-ea571c-pr6497.zip) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/9/) | 2025-04-07 16:21:23 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250407-161550-ea571c-pr6497.zip) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/9/) | 2025-04-07 16:21:46 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250407-161548-ea571c-pr6497.zip) |
| :heavy_multiplication_x: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/9/) | 2025-04-07 16:23:55 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/9//consoleText) |
| :heavy_check_mark: | ea571c05 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/9/) | 2025-04-07 16:51:25 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/10/) | 2025-04-08 20:24:42 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250408-202115-e20db1-pr6497.aar) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/10/) | 2025-04-08 20:24:50 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250408-202117-e20db1-pr6497.zip) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/10/) | 2025-04-08 20:25:10 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250408-202126-e20db1-pr6497.zip) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/10/) | 2025-04-08 20:26:32 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250408-202121-e20db1-pr6497.zip) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/10/) | 2025-04-08 20:26:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250408-202124-e20db1-pr6497.zip) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/10/) | 2025-04-08 20:27:58 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250408-202119-e20db1-pr6497.zip) |
| :heavy_multiplication_x: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/10/) | 2025-04-08 20:30:31 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/10//consoleText) |
| :heavy_check_mark: | e20db1b8 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/10/) | 2025-04-08 20:56:25 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/11/) | 2025-04-09 13:13:15 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250409-130948-5d7fba-pr6497.zip) |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/11/) | 2025-04-09 13:13:22 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250409-130947-5d7fba-pr6497.aar) |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/11/) | 2025-04-09 13:13:22 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250409-130958-5d7fba-pr6497.zip) |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/11/) | 2025-04-09 13:15:09 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250409-130952-5d7fba-pr6497.zip) |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/11/) | 2025-04-09 13:15:27 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250409-130954-5d7fba-pr6497.zip) |
| :heavy_check_mark: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/11/) | 2025-04-09 13:16:34 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250409-130950-5d7fba-pr6497.zip) |
| :heavy_multiplication_x: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/11/) | 2025-04-09 13:18:37 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/11//consoleText) |
| :heavy_multiplication_x: | 5d7fba27 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/11/) | 2025-04-09 13:46:37 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/12/) | 2025-04-10 10:20:57 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250410-101752-40956d-pr6497.aar) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/12/) | 2025-04-10 10:21:22 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250410-101801-40956d-pr6497.zip) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/12/) | 2025-04-10 10:21:23 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250410-101753-40956d-pr6497.zip) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/12/) | 2025-04-10 10:23:09 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250410-101756-40956d-pr6497.zip) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/12/) | 2025-04-10 10:23:24 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250410-101757-40956d-pr6497.zip) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/12/) | 2025-04-10 10:23:50 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250410-101754-40956d-pr6497.zip) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/12/) | 2025-04-10 10:26:09 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/12//consoleText) |
| :heavy_check_mark: | 40956dc9 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/12/) | 2025-04-10 10:52:37 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/12//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/13/) | 2025-04-10 14:59:25 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250410-145616-edba13-pr6497.aar) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/13/) | 2025-04-10 14:59:45 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250410-145619-edba13-pr6497.zip) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/13/) | 2025-04-10 15:00:08 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250410-145628-edba13-pr6497.zip) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/13/) | 2025-04-10 15:01:35 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250410-145621-edba13-pr6497.zip) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/13/) | 2025-04-10 15:01:54 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250410-145623-edba13-pr6497.zip) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/13/) | 2025-04-10 15:03:49 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250410-145619-edba13-pr6497.zip) |
| :heavy_check_mark: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/13/) | 2025-04-10 15:05:45 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/13//consoleText) |
| :heavy_multiplication_x: | edba13f7 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/13/) | 2025-04-10 15:31:57 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/13//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6497/14/) | 2025-04-14 07:48:23 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250414-074503-c803a5-pr6497.aar) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6497/14/) | 2025-04-14 07:48:59 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250414-074504-c803a5-pr6497.zip) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6497/14/) | 2025-04-14 07:49:08 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250414-074515-c803a5-pr6497.zip) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6497/14/) | 2025-04-14 07:50:42 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250414-074510-c803a5-pr6497.zip) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6497/14/) | 2025-04-14 07:50:57 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250414-074508-c803a5-pr6497.zip) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6497/14/) | 2025-04-14 07:51:10 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250414-074505-c803a5-pr6497.zip) |
| :heavy_multiplication_x: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/14/) | 2025-04-14 07:54:03 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6497/14//consoleText) |
| :heavy_check_mark: | c803a56c | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/14/) | 2025-04-14 08:20:33 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6497/14//consoleText) |

alwx commented:
*Makes a note that `getSuggestedRoute` also needs to be updated because of how prices are returned* (cc @shivekkhurana)

Haven't checked the details of the implementation but the API looks ok.

vkjr commented:
@shivekkhurana, I think integration of this PR deserves a separate issue because potential surface of changes in mobile code seems significant.

saledjenic committed:
chore(wallet)!: code updated to use token key (coingecko token's id) instead of symbol

What's changed:
- `marketManager` works with coingecko provider only (for now, until we add coin-gecko to the proxy), `cryptoCompare` and
`cryptoCompareProxy` providers are removed as options.
- cryptocompare providers's public functions marked as deprecated

Changed endpoints (all of them accept list of grouped tokens key/s (which is actually coingecko tokens' id) instead list of symbol/s):
- `wallet_fetchPrices`
- `wallet_fetchMarketValues`
- `wallet_getHourlyMarketValues`
- `wallet_getDailyMarketValues`
- `wallet_fetchTokenDetails`
- `wallet_fetchAllCurrencyFormats`
- `wallet_getCachedCurrencyFormats`

Changed signals:
- `UpdatedPrices` of `wallet.suggested.routes` signal returns price/s per grouped tokens key/s

saledjenic committed:
chore(wallet)_: don't fetch tokens from market provider/s

Instead of fetching the tokens from the market provider/s, the app should use tokens that are
available from the `token` package.

saledjenic committed:
chore(wallet)_: cryptocompare market provider related code removed

saledjenic committed:
chore(wallet)!: balance related endpoints updated to use tokens group key/s instead of a symbol

`IsNative` function of the `Token` type evaluates `GroupKey` instead of `Symbol`.

Affected calls:
- `GetBalanceHistory`
- `GetBalanceHistoryRange`

Updated fields:
- `NativeGroupedKey` added to the `Network` type
- `RouteInputParams.TokenID` renamed to `RouteInputParams.FromGroupedTokensKey`
- `RouteInputParams.ToTokenID` renamed to `RouteInputParams.ToGroupedTokensKey`
- `SendDetails.FromToken` renamed to `SendDetails.FromGroupedTokensKey`
- `SendDetails.ToToken` renamed to `SendDetails.ToGroupedTokensKey`

saledjenic commented:
Closing since it's not relevant anymore.
